lib/postgres.go

```diff
@@ -1288,7 +1289,7 @@ func (postgres *Postgres) UpsertBlockTx(tx *pg.Tx, blockNode *BlockNode) error {
 
 		TxMerkleRoot: blockNode.Header.TransactionMerkleRoot,
 		Version:      blockNode.Header.Version,
-		Timestamp:    blockNode.Header.TstampSecs,
+		Timestamp:    blockNode.Header.TstampNanoSecs,
```

Comment:
No need for action here, just note that all this code is deprecated in favor of state syncer, which has a similar logic in it that we can update anytime we want.

Reply:
Fully expecting to deprecate this once the state syncer work is merged. This change is here for completeness so the code still behaves as expected